lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -237,52 +241,38 @@ functions:
             year: 2018
             product: wofs_albers
             tag: '2018'
-  execute_stacker:
-    handler: handler.execute_ssh_command
-    environment:
-      cmd: 'execute_stacker --dea-module ${self:provider.environment.DEA_MODULE}
-                            --queue ${self:provider.environment.QUEUE}
-                            --project ${self:provider.environment.PROJECT}
-                            --year <%= year %>
-                            --app-config <%= appconfig %>'
-    events:
-      - schedule:
-          rate: cron(30 04 * * ? *) # Run daily, at 04:30:00 am UTC time
-          enabled: false
-          input:
-            year: 2018
-            appconfig: ls8_nbar_albers.yaml
-      - schedule:
-          rate: cron(45 04 * * ? *) # Run daily, at 04:45:00 am UTC time
-          enabled: false
-          input:
-            year: 2018
-            appconfig: ls8_nbart_albers.yaml
-      - schedule:
-          rate: cron(00 05 * * ? *) # Run daily, at 05:00:00 am UTC time
-          enabled: false
-          input:
-            year: 2018
-            appconfig: ls8_pq_albers.yaml
   execute_coherence:
     # execute_coherence shall not be used on nbar/nbart/pq albers products
     handler: handler.execute_ssh_command
+    description: Execute dea-coherence tool to check the database inconsistencies (duplicates, siblings, locationless)
     environment:
       cmd: 'execute_coherence --dea-module ${self:provider.environment.DEA_MODULE}
                               --queue ${self:provider.environment.QUEUE}
                               --project ${self:provider.environment.PROJECT}
-                              --stage ${self:custom.Stage}
-                              --timerange <%= timerange %>
-                              --product <%= product %>'
+                              --timequery <%= timequery %>'
     events:
       - schedule:
-          rate: cron(10 08 ? * THU *) # Run every Thursday, at 08:10:00 am UTC time
-          enabled: false
+          rate: cron(00 09 ? * * *) # Run every day, at 07:00 pm Canberra time
+          enabled: true
           input:
-            timerange: '2018-01-01 < time < 2018-12-31'
-            product: ls8_fc_albers
+            timequery: "time in 2018"
+  execute_notify_on_error_ds:
+    handler: handler.execute_ssh_command
+    description: Detect CSV file containing discrepencies in database, upload CSV to S3 and notify (Slack and Email)
+    environment:
+      cmd: 'execute_notify_on_error_ds --dea-module ${self:provider.environment.DEA_MODULE}
+                                       --awsprofile <%= awsprofile %>
+                                       --recemail <%= recemail %>'
```

Comment:
The quick and dirty way of removing the WebHook URL from the code repository, is to add an argument `--webhook-url` here, put the hook URL in SSM, and use `--webhook-url ${ssm:nameOfKey}`.

See https://serverless.com/blog/serverless-secrets-api-keys/ for more details.

The drawback of this is that it'll be hard coded into the deployment configuration instead of dynamically retrieved when running the lambda, but it's so much simpler than modifying our lambda code that it's worth it.

Reply:
Implemented in `PR#76`